### ISSUE TITLE
conflicting json imports in work/views

### ISSUE THREAD

**Issue by bhaugen** [FreedomCoop/valuenetwork#142]
Up top in the intro imports:
```import json as simplejson```

Later on, amongst some fobi code,
```import simplejson as json```

@XaviP I think you set up the first one in valueaccounting/views and @bum2 set up the second one when integrating fobi.

I have no idea what these are actually doing and what the effects may be, but it doesn't seem right...so I am tagging it as a bug until somebody tells me different.

**Comment by XaviP** [comment]
Thanks @bhaugen. When we've made the upgrading to django, the use of simplejson was depreciated and removed, for the use of native json python library. But to change the import as "json" has some conflicts with a variable called json inside the code. So, I decided to import json as simplejson to easy the upgrade.
I don't know if this concrete file uses json as a variable, but we can check it and if not needed we can import json as is, and simplejson as is, if you think it's necessary.

**Comment by XaviP** [comment]
> It assumes simplejson exists.

Yes, simplejson is a dependency of django-fobi, so it's installed.
